exp/views/study.py

```diff
@@ -330,6 +332,23 @@ class StudyResponsesList(LoginRequiredMixin, PermissionRequiredMixin, generic.De
     permission_required = 'studies.can_view_study_responses'
     raise_exception = True
 
+    def get_context_data(self, **kwargs):
+        """
+        In addition to the study, adds several items to the context dictionary.  Study results
+        are paginated.
+        """
+        context = super().get_context_data(**kwargs)
+        orderby = self.request.GET.get('sort', 'id')
+        page = self.request.GET.get('page', None)
+        study = context['study']
+        responses = study.responses.filter(completed=True).order_by(orderby) if orderby else study.responses.all()
+        context['responses'] = self.paginated_queryset(responses, page, 10)
+        context['response_data'] = self.build_responses(context['responses'])
```

Comment:
I worry not having this in a separate view might slow things down. Doing it every time someone lists study responses in multiple formats even though they might only want one of them. It's pretty well broken down into pieces so we can split it up later if we need to.